doc/articles/controls/WebView.md

```diff
@@ -168,6 +168,22 @@ public App()
 >
 > This feature will only work for security reasons when the application runs in Debug mode.
 
+## X11 specifics
+
+In order to use WebView2 on Linux, you'll need to install libwebkit and libgtk3-0:
```

Comment:
libwebkit2gtk specifically, because there's also an incompatible libwebkitgtk2 (notice that it's gtk2 not 2gtk)